File: goorm.io/1_2.c
#include <stdio.h>

int main() {
	long long int a;
	scanf("%lld", &a);
	
	int ans = 0;
	while(1) {
		if (a == 1) break;
		if (a % 2 == 1) {
			ans = 1;
			break;
		}
		else {
			a = a / 2;
			if (a == 0) break;
		}
	}
	
	if (!ans) printf("Yes\n");
	else printf("No\n");
	
	return 0;
}

File: goorm.io/1_2.py
a = int(input())

ans = 0
while True:
	if a == 1 or a == 0:
		break
	elif a % 2 == 1:
		ans = 1
		break
	else:
		a /= 2

if ans == 0:
	print('Yes')
else:
	print('No')